src/view/grants.njk

```diff
@@ -12,69 +12,68 @@
 
             <h3>Use of trademarks</h3>
-            <p>Last but not least, the Trustlines trademarks owned by the Foundation can be used by projects building on top of the Trustlines protocol in accordance with its Trustlines Trademark Guidelines. This way your project can already benefit from the popularity of the Trustlines protocol starting day one.</p>
-            <p>For example, this trademark will give you better access to the existing Trustlines community and thus increase your reach to promote your project to potential users.</p>
-            <p>Please find more information about our trademarks and how to work with them <a href="https://docs.trustlines.network/docs/foundation/trademarks.html" target="_blank">here</a>.</p>
+            <p>Last but not least, the Foundation's Trustlines trademarks can be used by projects building on top of the Trustlines Protocol following the <a href="https://docs.trustlines.network/docs/foundation/trademarks.html" target="_blank">Trustlines Trademark Guidelines</a>. This way, your project can already benefit from the popularity of the Trustlines Protocol starting day one.</p>
+            <p>For example, this trademark will give you better access to the existing Trustlines community and increase your reach to promote your project to potential users.</p>
 
             <h2>5. How to apply</h2>
-            <p>As soon as  applications will open, we will be waiting for your submission. As of now, we are not planning to close the application window anytime soon. In short, you can apply anytime.</p>
+            <p>As soon as  applications open, we will be waiting for your submission. As of now, we are not planning to close the application window anytime soon. In short, you can apply anytime.</p>
             <p>Once we receive your application, we will start the evaluation process. The evaluation process includes an initial screening of your application, followed by an in-depth review and potentially also an interview if necessary.</p>
-            <p>The duration of this process is highly dependent on the level of detail of your application and the scope of your project in general. Thus we ask for your understanding that we are not able to provide any specific timeframe in which the evaluation process will take place.</p>
-            <p>Please be aware that we might contact you for additional information throughout the whole process in case we are missing any, so we kindly ask you to be available.</p>
-            <p>You can apply via <a href="https://docs.google.com/forms/d/e/1FAIpQLSdSGkkafn3jDvFmv8S_FwWVoSkGXc6UGMDjTKw6Maoibcz5Jg/viewform">Google Forms.</a></p>
+            <p>This process's duration depends on your application's level of detail and the scope of your project in general. Thus we ask for your understanding that we cannot provide any specific timeframe in which the evaluation process will occur.</p>
```

Comment:
I'd stick to 'The duration of this process' and 'the level detail of your application'